README.md

```diff
@@ -621,7 +619,10 @@ echo "Operation status: ".$status->getStatus()."<br />";
 ```
 
 ##Media Services
-
+> **Note**
+> 
+>The current Media Services API only works with PHP 5.5 and above.
```

Comment:
@yaqiyang, [`empty()`](http://php.net/manual/en/function.empty.php) is part of the PHP core since the version 4. The difference at versions previous to PHP 5.5 is it only accepted variables and no other expressions.
>
Note:
Prior to PHP 5.5, empty() only supports variables; anything else will result in a parse error. In other words, the following will not work: empty(trim($name)). Instead, use trim($name) == false.